src/ua/ua_test.cpp

```diff
@@ -149,14 +152,54 @@ void certificateFile() {
 }
 
 void testWhoAmI(){
-  cout << "  Current User: ";
+  cout << "Current User: ";
   try {
     JSON res = systemWhoami(string("{}"), false);
     cout  << res["id"].get<string>() << endl;
   } catch(DXAPIError &e) {
     cout << "Error contacting the api: " << e.what() << endl;
   } catch (DXConnectionError &e) {
     cout << "Error contacting the api: " << e.what() << endl;
+  } catch (JSONException &e) {
+    cout << "Error contacting the api " << e.what() << endl;
+  } catch (...) {
+    cout << "Error contacting the api." << endl;
```

Comment:
Pedantically minor: there is a period at the end of this default message but are there for the other exceptions?

Reply:
No, the e.what() does not have a period at the end. Should I add one to all?